docs/yubikey_setup.rst

```diff
@@ -67,7 +67,7 @@ If you have just installed SecureDrop, you will need to add the first
 admin user to the Journalist Interface with ``manage.py``. ``cd`` to the
 ``SECUREDROP_ROOT``, which is ``/vagrant/securedrop`` in development and
 ``/var/www/securedrop`` in production. Run ``./manage.py add-admin``.
-Fill in the username and password prompts. When it asks "Is this admin
+Fill in the username prompt. When it asks "Is this admin
```

Comment:
Do you think we should add an extra reminder here for them to write down the diceware password?

Reply:
Good call @heartsucker - added, squashed, repushed :)